src/GraphQl/Resolver/Factory/ResolverFactory.php

```diff
@@ -32,17 +32,19 @@ public function __construct(
     public function __invoke(string $resourceClass = null, string $rootClass = null, Operation $operation = null): callable
     {
         return function (?array $source, array $args, $context, ResolveInfo $info) use ($resourceClass, $rootClass, $operation) {
+            // special treatment for nested resources without a resolver/provider
```

Comment:
mhh this line is from the other PR right?

Reply:
Yes. It was introduced by other PR. I moved this block one level up, though it also gets called when nested collection is `null`

Reply:
I created another PR based on 3.2 that only contains the bugfix relevant for 3.2. This one here also contains the `null`-fix for the revently introduced "nested collection" feature